### PR TITLE
CAS GA changes to RA docs

### PR DESCRIPTION
**Pull request by tashian**
- [ ] Add updated screenshots that include pool ID placeholder in the Marketplace app setup screens
- [x] Actually test this on GCP

From Mariano:

If you want to try CAS v1, you can use the `casv1` branch on both cli and certificates
`step ca init` ask you if you want to use a GCS bucket or use a managed one. Well, you have to leave it empty to use a managed one. But I haven’t been able to figure out the proper permissions for a custom one

To init a new RA on GCP (without using the marketplace application):

`step ca init --ra cloudcas --credentials-file fooo.json`

### CODE REVIEW

**Comment by devadvocado** [comment]
@tashian could you give this a try? thanks!

File: registration-authorities/acme-for-cas.mdx [hunk 1]
```diff
@@ -37,53 +37,69 @@ We will deploy the RA as a VM instance that issues X.509 Certificates for intern
...
+
+To create, run:
+
+  ```shell
+   gcloud privateca roots create example-root-ca \
+       --pool example-ca-pool \
        --location us-west1 \
        --subject "CN=Example Root CA, O=Example LLC" \
-       --key-algorithm ec-p256-sha256 \
-       --max-chain-length 1
```
Comment (tashian) [inline registration-authorities/acme-for-cas.mdx]: is there a new way to constrain the path length?

File: registration-authorities/acme-for-cas.mdx [hunk 2]
```diff
@@ -37,53 +37,69 @@ We will deploy the RA as a VM instance that issues X.509 Certificates for intern
...
+   gcloud privateca subordinates create example-intermediate-ca \
+       --pool example-ca-pool \
        --location us-west1 \
-       --issuer example-root-ca \
+       --issuer-ca example-root-ca \
+       --issuer-pool example-ca-pool \
        --issuer-location us-west1 \
        --subject "CN=Example Intermediate CA, O=Example LLC" \
-       --key-algorithm ec-p256-sha256 \
-       --reusable-config "subordinate-server-tls-pathlen-0"
```
Comment (tashian) [inline registration-authorities/acme-for-cas.mdx:73]: Is there a new way to constrain the path length for the intermediate?

File: registration-authorities/acme-for-cas.mdx [hunk 3]
```diff
@@ -187,7 +203,7 @@ new ACME RA for CAS, but any popular ACME client should work.
   $ export PROJECT_ID=$(curl http://metadata.google.internal/computeMetadata/v1/project/project-id --silent --fail -H "Metadata-Flavor: Google")
 
   # Get a certificate!
-  $ sudo REQUESTS_CA_BUNDLE=/tmp/root_ca.crt certbot certonly -n --standalone -d $HOST --server https://acme-registration-authority-for-gcp-cas-1-vm.c.$PROJECT_ID.internal/acme/acme/directory --agree-tos --email tester@smallstep.com
+  $ sudo REQUESTS_CA_BUNDLE=/tmp/root_ca.crt certbot certonly -n --standalone -d $HOST --server https://acme-registration-authority-for-gcp-c-1-vm.c.$PROJECT_ID.internal/acme/acme/directory --agree-tos --email tester@smallstep.com
```
Comment (tashian) [inline registration-authorities/acme-for-cas.mdx:229]: did u mean to make this change?

**Review by tashian — COMMENTED** [review]
@devadvocado I added a couple comments.
I'm not sure if I'm set up to test this, but if you want to screen share today and step through it together, let me know!

File: registration-authorities/acme-for-cas.mdx (hunk 3 above)
Comment (devadvocado) [reply registration-authorities/acme-for-cas.mdx:229]: Yea, the server url is now `...-gcp-c-...`

**Pull request merged**